scripts/main.js

```diff
@@ -31,17 +31,18 @@ $showNewAppFormButton.on('click', function () {
 $newAppForm.on('submit', function (event) {
   event.preventDefault()
 
-  // create app array for exisiting apps
   var app = {
     name: $newAppName.val()
   }
-  if (app.name) {
-    Apps.add(app)
 
-      .then(function (app) {
-        setRoute(app.id)
-      })
+  if (!app.name) {
```

Comment:
This is just a code style that I personally like. I try to avoid nesting code and especially try to avoid `else` as much as possible, and one way to achieve it is to return early. Here is some info on that code style: http://blog.timoxley.com/post/47041269194/avoid-else-return-early